fhirAssembly/README.md

```diff
@@ -250,7 +250,49 @@ the issue.
 
 ![SimplePatientError.png](docs/images/SimplePatientError.png)
 
+
+# Development using this Git Repo
+
 ## Building the FHIR Assembly
 
-TBD
+To build the FHIR assembly, use the `assemble` task for Gradle.  That is, run the command
+
+`./gradlew fhirAssembly:assemble`
+
+This will produce the 'com.vantiq.fhir.fhirConnection.zip' file in the project's `build/distribution` directory.
+
+Once that is done, you can import the assembly projects to a Vantiq namespace, and, from there, publish these
+assemblies to a catalog.  The steps involved are as follows.
+
+## Importing Assemblies
+
+This can be done manually (using the Vantiq IDE to import the generated projects).
+
+However, if importing many projects, use `../gradlew importAssemblies -PfhirAssembliesProfile=<profileName>`
+
+where `<profileName>` is the name of a Vantiq CLI profile that will connect to your publishing namespace.  Other
+gradle properties that are used are based on that profile name, and are as follows.
+
+* **<profileName>_fhirAssembliesVantiq** -- command to be used as the Vantiq CLI.  Defaults to `vantiq`.
```

Comment:
profilename does not show up in the final markdown. 
need to escape the > there
Same thing in usage at line 287 & 293
```suggestion
* **<profileName\>_fhirAssembliesVantiq** -- command to be used as the Vantiq CLI.  Defaults to `vantiq`.
```